lib/active_fedora/solr_query_builder.rb

```diff
@@ -11,28 +11,29 @@ def construct_query_for_ids(id_array)
         "{!terms f=#{ActiveFedora.id_field}}#{ids.join(',')}"
       end
 
-      # Create a query with a clause for each key, value
+      # Create a term query with a clause for each key, value
       # @param [Hash, Array<Array<String>>] field_pairs key is the predicate, value is the target_uri
       # @param [String] join_with ('AND') the value we're joining the clauses with
       # @example
-      #   construct_query_for_rel [[:has_model, "info:fedora/afmodel:ComplexCollection"], [:has_model, "info:fedora/afmodel:ActiveFedora_Base"]], 'OR'
-      #   # => _query_:"{!field f=has_model_ssim}info:fedora/afmodel:ComplexCollection" OR _query_:"{!field f=has_model_ssim}info:fedora/afmodel:ActiveFedora_Base"
+      #   construct_query_for_rel [[:has_model, "ComplexCollection"], [:has_model, "ActiveFedora_Base"]], 'OR'
+      #   # => _query_:"{!term f=has_model_ssim}ComplexCollection" OR _query_:"{!term f=has_model_ssim}ActiveFedora_Base"
       #
       #   construct_query_for_rel [[Book._reflect_on_association(:library), "foo/bar/baz"]]
       def construct_query_for_rel(field_pairs, join_with = default_join_with)
         field_pairs = field_pairs.to_a if field_pairs.is_a? Hash
-        construct_query(property_values_to_solr(field_pairs), join_with)
+        construct_query(property_values_to_solr(field_pairs), join_with, 'term'.freeze)
       end
 
       # Construct a solr query from a list of pairs (e.g. [field name, values])
       # @param [Array<Array>] field_pairs a list of pairs of property name and values
       # @param [String] join_with ('AND') the value we're joining the clauses with
+      # @param [String] type ('field') The type of query to run. Either 'term' or 'field'
       # @return [String] a solr query
       # @example
       #   construct_query([['library_id_ssim', '123'], ['owner_ssim', 'Fred']])
       #   # => "_query_:\"{!field f=library_id_ssim}123\" AND _query_:\"{!field f=owner_ssim}Fred\""
-      def construct_query(field_pairs, join_with = default_join_with)
-        clauses = pairs_to_clauses(field_pairs)
+      def construct_query(field_pairs, join_with = default_join_with, type = 'field'.freeze)
```

Comment:
Curious about using freeze on an optional parameter value. Should that be a constant? What is the expected behavior?